Modules/Quest/QuestEventHandler.lua

```diff
@@ -1,9 +1,13 @@
 ---@class QuestEventHandler
 local QuestEventHandler = QuestieLoader:CreateModule("QuestEventHandler")
+---@class QuestEventHandlerPrivate
 local _QuestEventHandler = QuestEventHandler.private
+
 local _QuestLogUpdateQueue = {} -- Helper module
 local questLogUpdateQueue = {} -- The actual queue
 
+---@type QuestEventHandlerPrivate
+QuestEventHandler.private = QuestEventHandler.private or {}
```

Comment:
This should happen before `local _QuestEventHandler = QuestEventHandler.private` in line 4, shouldn't it?